client/src/Helpers/tokenIsValid.js

```diff
@@ -0,0 +1,6 @@
+export const tokenIsValid = (token) => {
+  if (typeof(token) !== "string" || token.length !== 155) {
```

Comment:
JWTs are not necessarily always of length 155, they can be more or less depending on the payload/secret.

Also, with `typeof` the brackets aren't needed. You can just do `typeof token`. 👍

Reply:
I'll remove the length-checking. Any ideas as to what I could check the token for before sending it on to the back-end?

Will remove brackets around `typeof`. Thanks for the heads-up!

Reply:
I can create a new `/verify` endpoint which you can hit and receive a boolean back. 👍